tests/app/idp/pyproject.toml

```diff
@@ -19,4 +19,4 @@ py-modules = []
 [tool.uv.sources]
 # connect to the django-oauth-toolkit package in the parent workspace
 # so changes to it are reflected here without reinstalling.
-django-oauth-toolkit = { editable = true, workspace = true }
+django-oauth-toolkit = { workspace = true }
```

Comment:
oh, this isn't directly related obviously. I was unable to even get this app to work at all without this change:

```
$ uv run tox
error: Failed to parse: `tests/app/idp/pyproject.toml`
  Caused by: TOML parse error at line 22, column 24
   |
22 | django-oauth-toolkit = { editable = true, workspace = true }
   |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
cannot specify both `workspace` and `editable`
```